pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -505,16 +499,13 @@ func (l LibvirtWraper) SetupLibvirt() (err error) {
 		return err
 	}
 
-	libvirdDConf, err := os.OpenFile(runtimeLibvirtdConfPath, os.O_APPEND|os.O_WRONLY, 0644)
-	if err != nil {
-		return err
-	}
-	defer util.CloseIOAndCheckErr(libvirdDConf, &err)
-
-	// Let libvirt log to stderr
-	_, err = libvirdDConf.WriteString("log_outputs = \"1:stderr\"\n")
```

Comment:
why is okay to remove this line and the one regarding cgroups? Where are the duplicate lines coming from in this case?

Reply:
It's in https://github.com/kubevirt/kubevirt/pull/6145/commits/8cf78f057360dd882a5b7dd40b7bdab357edceb6 commit message, we already have them in the shipped config file that we use to build the images.

Reply:
yup, got it now. i see we have these configs in the kubevirt/kubevirt repo even.